libs/domain/site/menu-item/menu-item.model.ts

```diff
@@ -0,0 +1,34 @@
+export enum SiteMenuItemVariation {
+  Navigation = 'navigation',
+}
+
+export interface SiteMenuItemContent {
+  text?: string;
+}
+
+export interface SiteMenuItemOptions {
+  /**
+   * url item should link to. This is the absolute url and looks like /account/overview.
+   */
+  url?: string;
```

Comment:
If I'm not mistaken the initial idea was to have single field for url providing with complex type: `url?: string | LinkOptions;`. raw ulr vs LinkOptions, similar to navigation-item